client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -70,299 +89,321 @@ export const RequestLineEdit = ({
   return (
     <Box display="flex" flexDirection="column" padding={2}>
       <Box display="flex" justifyContent="space-between">
-        <Box paddingLeft={4} paddingRight={7}>
-          {/* Left column content */}
-          <InputWithLabelRow
-            Input={
-              <NumericTextInput
-                width={INPUT_WIDTH}
-                value={draft?.itemStats.availableStockOnHand}
-                disabled
-                autoFocus
-              />
-            }
-            labelWidth={LABEL_WIDTH}
-            label={t('label.stock-on-hand')}
-            sx={{ marginBottom: 1 }}
-          />
-          {isProgram && extraFields && (
-            <>
-              <InputWithLabelRow
-                Input={
-                  <NumericTextInput
-                    width={INPUT_WIDTH}
-                    value={draft?.incomingUnits}
-                    disabled
-                  />
+        {isNew ? (
+          <Box width="100%">
+            <StockItemSearchInput
+              onChange={(newItem: ItemRowFragment | null) => {
+                if (newItem) {
+                  insert({
+                    id: FnUtils.generateUUID(),
+                    requisitionId: requisitionId,
+                    itemId: newItem.id,
+                  });
+                  navigate(buildItemEditRoute(requisitionNumber, newItem.id));
                 }
-                labelWidth={LABEL_WIDTH}
-                label={t('label.incoming-stock')}
-                sx={{ marginBottom: 1 }}
-              />
+              }}
+              openOnFocus={true}
+              extraFilter={item =>
+                !lines.some(line => line.item.id === item.id)
+              }
+            />
+          </Box>
+        ) : (
+          <>
+            <Box paddingLeft={4} paddingRight={7}>
+              {/* Left column content */}
               <InputWithLabelRow
                 Input={
                   <NumericTextInput
                     width={INPUT_WIDTH}
-                    value={draft?.outgoingUnits}
+                    value={draft?.itemStats.availableStockOnHand}
```

Comment:
Can you summarise what you were changing here? Looks like you've added some fields... and hidden more things behind `useConsumptionData` - what does that mean? lol

Reply:
Ah no fields were added, they were just shifted since the stock autocomplete was
added at the top~ and sorry, that's the wrong pref! It's supposed to be
showExtraFields